scanpy/preprocessing/simple.py

```diff
@@ -593,36 +593,36 @@ def pca(data, n_comps=None, zero_center=True, svd_solver='auto', random_state=0,
         logg.msg('reducing number of computed PCs to',
                n_comps, 'as dim of data is only', adata.n_vars, v=4)
 
-    adata_hvg = adata[:, adata.var['highly_variable']] if use_hvg and 'highly_variable' in adata.var.keys() else adata
+    _adata = adata[:, adata.var['highly_variable']] if use_hvg and 'highly_variable' in adata.var.keys() else adata
```

Comment:
OK, this name is fine!

Reply:
Just remembered that in other instances, we have the convention `adata_comp` if the passed AnnData is not the object on which the actual computation is carried out, so that would be nicer. Thank you!